apps/backend/src/mutations/InstrumentMutations.ts

```diff
@@ -475,14 +475,12 @@ export default class InstrumentMutations {
 
     let isXpress = false;
 
-    if (!isRejection(instrumentWithTechnique)) {
-      isXpress =
-        instrumentWithTechnique.length > 0 &&
-        instrumentWithTechnique.filter(
-          (instruments) => instruments.id === args.instrumentIds[0]
-        ).length > 0;
-    } else {
-      return instrumentWithTechnique;
+    if (instrumentWithTechnique && instrumentWithTechnique.length > 0) {

```

Comment:
Looks good, though this check shouldn't be needed as it will be an array of instruments or empty array. Using find() on an empty array will still work without error